tests/integration/modules/test_localemod.py

```diff
@@ -46,7 +47,7 @@ def test_gen_locale(self):
         locale = self.run_function('locale.get_locale')
         new_locale = _find_new_locale(locale)
         ret = self.run_function('locale.gen_locale', [new_locale])
-        self.assertTrue(ret)
```

Comment:
We should still assert against `ret`, we're testing against `gen_locale`.
Probably `get_locale` again using the new locale, and then assert against the error not being present.

Reply:
asserting against `ret` is useless.  It always succeeds, even if the command failed.

Reply:
Hmm, that suggests a bad implementation.
Not saying it should be addressed in this PR, at least for now.